catalog/app/containers/Bucket/CopyButton.js

```diff
@@ -0,0 +1,133 @@
+import * as React from 'react'
+import * as M from '@material-ui/core'
+import * as Lab from '@material-ui/lab'
+
+import * as AWS from 'utils/AWS'
+import { useData } from 'utils/Data'
+
+import * as requests from './requests'
+
+function MenuPlaceholder() {
+  const t = M.useTheme()
+
+  return (
+    <M.Box minWidth={t.spacing(22)}>
+      <M.MenuItem disabled>
+        <Lab.Skeleton height={t.spacing(6)} width="100%" />
+      </M.MenuItem>
+      <M.MenuItem disabled>
+        <Lab.Skeleton height={t.spacing(6)} width="100%" />
+      </M.MenuItem>
+      <M.MenuItem disabled>
+        <Lab.Skeleton height={t.spacing(6)} width="100%" />
+      </M.MenuItem>
+    </M.Box>
+  )
+}
+
+function MenuItem({ item, onClick }) {
+  return (
+    <M.MenuItem onClick={React.useCallback(() => onClick(item), [item, onClick])}>
+      <M.ListItemText primary={item.name} secondary={item.url} />
+    </M.MenuItem>
+  )
+}
+
+function SuccessorsSelect({ anchorEl, bucket, open, onChange, onClose }) {
+  const s3 = AWS.S3.use()
+  const data = useData(requests.workflowsList, { s3, bucket })
+
+  // FIXME: add documentation link
+  return (
+    <M.Menu anchorEl={anchorEl} onClose={onClose} open={open}>
+      {data.case({
+        Ok: ({ successors }) => (
+          <div>
```

Comment:
this div wrapper is unnecessary (and seems like it also causes the menu positioning bug, see the attachment; removing it fixes the bug), `MenuItem`s should be the direct children of the `Menu`

<img width="533" alt="Screenshot 2020-12-01 at 20 46 09" src="https://user-images.githubusercontent.com/122294/100763069-88901180-3416-11eb-973c-a5b1c01ea7b4.png">

Reply:
Yeah, thanks. When I saw it I didn't realize it's a bug, because behavior is similar to `M.Select` :)